.github/workflows/render-rmarkdown.yaml

```
@@ -14,24 +14,29 @@ jobs:
+      - uses: r-lib/actions/setup-pandoc@473c68190595b311a74f208fba61a8d8c0d4c247 # v2.11.1
 
-      - uses: r-lib/actions/setup-r@v2
+      - uses: r-lib/actions/setup-r@473c68190595b311a74f208fba61a8d8c0d4c247 # v2.11.1
 
-      - uses: r-lib/actions/setup-r-dependencies@v2
+      - uses: r-lib/actions/setup-r-dependencies@473c68190595b311a74f208fba61a8d8c0d4c247 # v2.11.1
         with:
           extra-packages: any::rmarkdown, local::.
 
       - name: Render README.Rmd
         run:  Rscript -e 'rmarkdown::render("README.Rmd")'
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)